### PR TITLE
feat: homepage-static-page-done

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Static page for Homepage
2. Homepage is using card component for categories card and summer-collection cards
3.  The card component is done, albeit it will be updated and refactored in future PRs
4. Know More in button is  summer-collection cards routing to Login page for now, but it will be redirected to product page with respective filter as per the card category in future PRs.

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      >
+        
+        <img
+          className="card-img"
+          src={props.img.src}
+          alt={props.img.alt}
+        />
+
+        {/* <CardImage src={props.img.src}
+          alt={props.img.alt}/> */}
```
Comment (ozairkhan2810) [inline src/components/Card/Card.jsx]: remove comment

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    </Link>
+};
+
+const Card = (props) => {
+  return (
+      <div
+        className={`card-shadow ${
+          props.variant === "vertical" ? "card-vetical" : "card-horizontal"
+        } ${props.overlayText ? "text-overlay-container" : ''}
+        ${props.className? props.className : ""}`}
```
Comment (ozairkhan2810) [inline src/components/Card/Card.jsx:26]: can remove, this is syntax right? (line 22)

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            {props.cardButton &&  
+            <Link
+      to={props.cardButton.link}
+      className={`card-btn link-btn ${
+        props.cardButton.variant === "primary" ? "button-primary" : "button-secondary"
+      }`}
+    >
+      {props.cardButton.info}
+    </Link>}
+    {/* <CardButton link={props.cardButton.link} info={props.cardButton.info} variant={props.cardButton.variant}/> */}
```
Comment (ozairkhan2810) [inline src/components/Card/Card.jsx]: remove comment

File: src/components/Card/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            src="./images/unisex.png"
+            alt="unisex-category"
+          />
+        </div>
+        <div class="gray-tile"></div>
+        <div class="text-overlay">UNISEX</div>
+      </div>
+    </div>
+  );
+};
```
Comment (ozairkhan2810) [inline src/components/Card/Card.jsx]: use props if you are importing (in line 62) or can remove

**Review by ozairkhan2810 — APPROVED** [review]
Few suggestions, rest looks good

File: src/components/Card/Card.jsx (hunk 1 above)
Comment (ankit9015) [reply src/components/Card/Card.jsx]: commented code is not working, it is there for future PRs

File: src/components/Card/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  return (
+    <Link
+      to={props.link}
+      className={`card-btn link-btn ${
+        props.variant === "primary" ? "button-primary" : "button-secondary"
+      }`}
+    >
+      {props.info}
+    </Link>)
+};
```
Comment (rkapoor10) [inline src/components/Card/Card.jsx:14]: liked this use of props

File: src/components/Card/Card.jsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+          alt={props.img.alt}/>
+        
+        
+        <div className="card-body">
+            {props.title && <h3 className="card-title">{props.title}</h3>}
+            {props.subtitle && <h6 className="card-title-sub ">{props.subtitle}</h6>}
+            {props.description && <p className="card-description">{props.description}</p>}
+
+            {props.cardButton &&  
+            <CardButton link={props.cardButton.link} info={props.cardButton.info} variant={props.cardButton.variant}/>
```
Comment (rkapoor10) [inline src/components/Card/Card.jsx:39]: you could have destructured props... ```const {title,subtitle} = props```

**Review by rkapoor10 — APPROVED** [review]
The overall code looks good to merge.

**Pull request merged**